.helper-scripts/timescaledb/generate_guc_overview.py

```diff
@@ -118,26 +147,41 @@ def prepare(content: str) -> dict:
 
     # Find all GUCs based on patterns and prepare them in a dict
     for pattern, val in TYPES.items():
+        # Run twice to find variants, e.g. 
+        # - DefineCustomStringVariable(MAKE_EXTOPTION(
+        # - DefineCustomStringVariable(/* name= */ MAKE_EXTOPTION(
         map.update(unwrap(re.findall(r"%s\(MAKE_EXTOPTION(.*?)\);" % pattern, content, re.DOTALL), val))
+        map.update(unwrap(re.findall(r"%s\(\/\* name= \*\/ MAKE_EXTOPTION(.*?)\);" % pattern, content, re.DOTALL), val))
 
     # TODO: find scopes
     # https://github.com/timescale/timescaledb/blob/2.19.x/src/guc.c#L797
+    # SCOPES
 
+    # print summary
+    summary = {}
+    for v in map.values():
+        if v["type"] not in summary.keys():
+            summary[v["type"]] = 0
+        summary[v["type"]] += 1
+    for k, v in summary.items():
+        logging.info("registered %d GUCs of type: %s" % (v, k))
 
     # Return dict with alphabetically sorted keys
     return {i: map[i] for i in sorted(map.keys())}
 
 """
 Render the GUCs to file
 """
-def render(gucs: dict, filename: str):
+def render(gucs: dict, filename: str, version: str):
     with open(filename, "w") as f:
-        f.write("| Name | Type | Default | Long Description |\n")
-        f.write("| -- | -- | -- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|\n")
+        f.write("| Name | Type | Default | -- | Long Description |\n")
```

Comment:
```suggestion
        f.write("| Name | Type | Default | -- | Description |\n")
```
Why do you want an empty column with no title?

Reply:
for boolean, it's always empty. For integer and real values, this shows the min & max values and for enum (eventually) the different options